src/store/blockchain/actions.ts

```diff
@@ -319,25 +315,14 @@ export const initWallet: ThunkCreator<Promise<any>> = () => {
                         userOrders: [],
                     }),
                 );
-                dispatch(setMarketTokens({ baseToken, quoteToken }));
-
                 // tslint:disable-next-line:no-floating-promises
-                dispatch(getOrderbookAndUserOrders());
-
+                dispatch(updateGasInfo());
                 // tslint:disable-next-line:no-floating-promises
-                dispatch(getAllCollectibles(ethAccount));
-
-                try {
-                    await dispatch(fetchMarkets());
-                    // For executing this method is necessary that the setMarkets method is already dispatched, otherwise it wont work (redux-thunk problem), so it's need to be dispatched here
-                    // tslint:disable-next-line:no-floating-promises
-                    dispatch(setConnectedUserNotifications(ethAccount, networkId));
-                    // tslint:disable-next-line:no-floating-promises
-                    dispatch(updateMarketPriceEther());
-                } catch (error) {
-                    // Relayer error
-                    logger.error('The fetch orders from the relayer failed', error);
-                }
+                dispatch(updateMarketPriceEther());
+                // Inits wallet based on the current app
+                currentRoute.includes(ERC20_APP_BASE_PATH)
```

Comment:
Pp: it looks like this check is being used multiple times and is kind of important. Should we put this inside a function or selector? Lets' check what the other guys think about it with their :+1: or :-1:

Reply:
Also, I personally like to us `if` for statements (conditions/code flow) and the leave the `?` operator for expressions (assignment, return value), but that's being me picky.

Reply:
Strong +1 to both things.

Reply:
It's done